src/api/helpers/formatEntityData.js

```diff
@@ -0,0 +1,386 @@
+			endDate: _.get(author, 'endDate', null),
+			ended: _.get(author, 'ended', null),
+			gender: _.get(author, 'gender.name', null),
+			type: _.get(author, 'authorType.label', null)
+		};
+}
+
+/**
+ * A function to extract the basic details of a Publisher ORM entity
+ * @function
+ * @param {object} publisher - a Publisher ORM entity
+ * @returns {object} an object containing the basic data of a publisher entity.
+ * Each property defaults to null. If publisher is null or undefined, returns null.
+ *
+ * @example
+ * 		const author = await orm.func.entity.getEntity(orm, 'Publisher', bbid, relations);
+ *		getPublisherBasicInfo(publisher);
+ *		/* => {
+			"area": "India",
+			"bbid": "e418874e-5684-4fe9-9d2d-1b7e5d43fd59",
+			"beginDate": "1943",
+			"defaultAlias": {
+				"aliasLanguage": "[Multiple languages]",
+				"name": "Bharati Bhawan",
+				"sortName": "Bhawan, Bharati"
+			},
+			"disambiguation": null,
+			"endDate": null,
+			"ended": false,
+			"type": "Publisher"
+		}
+ */
+
+export function getPublisherBasicInfo(publisher: object) {
+	return _.isNil(publisher) ? null :
+		{
+			area: _.get(publisher, 'area.name', null),
+			bbid: _.get(publisher, 'bbid', null),
+			beginDate: _.get(publisher, 'beginDate', null),
+			defaultAlias: getDefaultAlias(publisher),
+			disambiguation: _.get(publisher, 'disambiguation.comment', null),
+			endDate: _.get(publisher, 'endDate', null),
+			ended: _.get(publisher, 'ended', null),
+			type: _.get(publisher, 'publisherType.label', null)
+		};
+}
+
+/**
+ * getEntityAliases is a function to extract the list of aliases from ORM entity
+ *
+ * @param {object} entity - An ORM Entity
+ * @returns {object} an object conatining the bbid and list of aliases data of an ORM entity
+ * Each property of object defaults to null. If entity is null or undefined, returns null.
+ * @example
+ *		const edition = await orm.func.entity.getEntity(orm, 'Edition', bbid, relations);
+ *		getEntityAliases(edition);
+ *		/* => {
+			"aliases": [
+				{
+					"aliasLanguage": "English",
+					"name": "A Monster Calls",
+					"primary": true,
+					"sortName": "Monster Calls, A"
+				}
+			],
+			"bbid": "442ab642-985a-4957-9d61-8a1d9e82de1f"
+		}
+ */
+
+export function getEntityAliases(entity: object) {
+	return _.isNil(entity) ? null :
+		{
+			aliases: _.get(entity, 'aliasSet.aliases', []).map((alias) => ({
+				aliasLanguage: _.get(alias, 'language.name', null),
+				name: _.get(alias, 'name', null),
+				primary: _.get(alias, 'primary', null),
+				sortName: _.get(alias, 'sortName', null)
+			})),
+			bbid: _.get(entity, 'bbid', null)
+		};
+}
+
+/**
+ * getEntityIdentifiers is a function to extract the list of indentifiers from an ORM entity
+ *
+ * @param {object} entity - An ORM entity
+ * @returns {object} an object containing the bbid and list of identifier data of an entity
+ * Each property of object defaults to null. If entity object is null or undefined, returns null.
```

Comment:
'Each property of object defaults to null' should be replaced with "Returns an empty array if no identifiers were found"